Example/Controllers/SwiftUI/Helper/MiniAppViewMessageDelegator.swift

```diff
@@ -254,3 +263,9 @@ extension MiniAppViewMessageDelegator: CLLocationManagerDelegate {
         }
     }
 }
+
+extension MiniAppViewMessageDelegator {
+    func sendJsonStringToMiniApp(jsonString: String) {
+        MiniApp.shared().sendJsonStringToMiniApp(withId: miniAppId, miniAppVersion: miniAppVersion ?? "", jsonString: jsonString)
```

Comment:
We shouldn't support this. Host app can't use our old singleton patterns to send message to Mini App

Reply:
This is now removed in the latest commits.